lib/pipeline.js

```diff
@@ -191,6 +192,54 @@ class PipelineModel extends BaseModel {
     }
 
     /**
+     * Fetch all the active jobs that belong to this pipeline
+     * Sorted by workflow order, followed by PR jobs
+     * @return {[type]} [description]
+     */
+    getActiveJobs() {
```

Comment:
Nothing get passed in so yes I can make it a getter.
Or we can make this take in archived = true or false and pagination. 

What would you prefer?